public/app/plugins/panel/barchart/utils.ts

```diff
@@ -82,7 +82,7 @@ export const preparePlotConfigBuilder: UPlotConfigPrepFn<BarChartOptionsEX> = ({
 }) => {
   const builder = new UPlotConfigBuilder();
   const defaultValueFormatter = (seriesIdx: number, value: any) => {
-    return shortenValue(formattedValueToString(frame.fields[seriesIdx].display!(value)), xTickLabelMaxLength);
```

Comment:
this looks like it was a mistake. we should not apply tick label length limits to the bar values. removing them causes them to be resized down instead of uselessly truncated:

![image](https://user-images.githubusercontent.com/43234/204155689-68dc7250-8e06-4a52-95b7-6ba36b1cc8df.png)